thirdeye/thirdeye-frontend/app/mirage/endpoints/rootcause.js

```diff
@@ -136,6 +136,8 @@ export default function(server) {
    * should return mocked data
    */
   server.get('/rootcause/metric/aggregate', () => {});
```

Comment:
We should look into extracting these consts of these urls into a file under `utils/api` later.